services/noteservice/noteservice.go

```diff
@@ -0,0 +1,47 @@
+/*
+Package noteservice handles interactions with database layer.
+*/
+package noteservice
+
+import (
+	"errors"
+
+	"github.com/atmiguel/cerealnotes/databaseutil"
+	"github.com/atmiguel/cerealnotes/models"
+)
+
+var NoteIdNotSet error = errors.New("The NoteId was not set")
+
+func StoreNewNote(
+	note *models.Note,
+) error {
+
+	id, err := databaseutil.StoreNewNote(int64(note.AuthorId), note.Content, note.CreationTime)
+	if err != nil {
+		return err
+	}
+
+	note.Id = id
```

Comment:
This line here makes me feel a bit better that we created a `<object>service` and `databaseutil` organization. Rather than just all of the stuff in one place.

Reply:
Why's that?